From 4ec6017908d6e990f726372a567678ca33888404 Mon Sep 17 00:00:00 2001
From: Edwin Pers <pers.edwin@gmail.com>
Date: Wed, 15 Oct 2025 11:53:00 -0400
Subject: [PATCH] drop to cu126 in deps

---
 pins.json      | 4 ++--
 pyproject.toml | 6 +++---
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/pins.json b/pins.json
index 122e6c8e0e..90043e9a3b 100644
--- a/pins.json
+++ b/pins.json
@@ -2,12 +2,12 @@
   "python": "3.12",
   "torchIndexUrl": {
     "win32": {
-      "cuda": "https://download.pytorch.org/whl/cu128"
+      "cuda": "https://download.pytorch.org/whl/cu126"
     },
     "linux": {
       "cpu": "https://download.pytorch.org/whl/cpu",
       "rocm": "https://download.pytorch.org/whl/rocm6.3",
-      "cuda": "https://download.pytorch.org/whl/cu128"
+      "cuda": "https://download.pytorch.org/whl/cu126"
     },
     "darwin": {}
   }
diff --git a/pyproject.toml b/pyproject.toml
index ac3d88b364..774e43c129 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -46,7 +46,7 @@ dependencies = [
   "safetensors",
   "sentencepiece==0.2.0",                 # 0.2.1 coredumps windows when loading t5 tokenizer
   "spandrel",
-  "torch~=2.7.0",                         # torch and related dependencies are loosely pinned, will respect requirement of `diffusers[torch]`
+  "torch~=2.6.0",                         # torch and related dependencies are loosely pinned, will respect requirement of `diffusers[torch]`
   "torchsde",                             # diffusers needs this for SDE solvers, but it is not an explicit dep of diffusers
   "torchvision",
   "transformers>=4.56.0",
@@ -87,7 +87,7 @@ dependencies = [
 ]
 
 "cpu" = ["torch==2.7.1+cpu", "torchvision==0.22.1+cpu"]
-"cuda" = ["torch==2.7.1+cu128", "torchvision==0.22.1+cu128"]
+"cuda" = ["torch==2.6.0+cu126", "torchvision==0.21.0+cu126"]
 "rocm" = [
   "torch==2.7.1+rocm6.3",
   "torchvision==0.22.1+rocm6.3",
@@ -149,7 +149,7 @@ explicit = true
 
 [[tool.uv.index]]
 name = "torch-cuda"
-url = "https://download.pytorch.org/whl/cu128"
+url = "https://download.pytorch.org/whl/cu126"
 explicit = true
 
 [[tool.uv.index]]
-- 
2.51.0

